src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2

```diff
@@ -207,6 +207,7 @@ module "openvpn" {
 module "Users" {
   source = "./modules/iam"
```

Comment:
@prem-fissionhq pointed out online that server.iam would be a better place for these changes, so I'll review again after that